app/src/main/java/org/oppia/app/topic/play/TopicPlayFragmentPresenter.kt

```diff
@@ -25,16 +28,28 @@ class TopicPlayFragmentPresenter @Inject constructor(
   activity: AppCompatActivity,
   private val fragment: Fragment,
   private val logger: Logger,
+  private val explorationDataController: ExplorationDataController,
   private val topicController: TopicController
-) : StorySummarySelector {
-
+) : StorySummarySelector, ChapterSummarySelector {
+  private val routeToExplorationListener = activity as RouteToExplorationListener
   private val routeToStoryListener = activity as RouteToStoryListener
 
+  private var currentExpandedChapterListIndex = NO_INDEX
+
   private lateinit var binding: TopicPlayFragmentBinding
 
-  fun handleCreateView(inflater: LayoutInflater, container: ViewGroup?): View? {
-    binding = TopicPlayFragmentBinding.inflate(inflater, container, /* attachToRoot= */ false)
+  private lateinit var expandedChapterListIndexListener: ExpandedChapterListIndexListener
 
+  fun handleCreateView(
+    inflater: LayoutInflater,
+    container: ViewGroup?,
+    currentExpandedChapterListIndex: Int,
+    expandedChapterListIndexListener: ExpandedChapterListIndexListener
+  ): View? {
+    this.currentExpandedChapterListIndex = currentExpandedChapterListIndex
+    this.expandedChapterListIndexListener = expandedChapterListIndexListener
+
```

Comment:
Is this empty line part of codestyle?

Reply:
This seems reasonable. We shouldn't have more than 1 empty newline anywhere, but in most other places it's up to the preference of the developer: https://google.github.io/styleguide/javaguide.html#s4.6.1-vertical-whitespace.